packages/flutter_tools/lib/src/flx.dart

```diff
@@ -115,7 +115,13 @@ Map<_Asset, List<_Asset>> _parseAssets(
 
       List<FileSystemEntity> files = assetDir.listSync(recursive: true);
 
+      final String buildDirPath = path.absolute(path.join(assetBase, 'build'));
```

Comment:
That defaults to `build/flx`. We do want to exclude that dir - to avoid recursion - but I think we want to exclude the contents of the full `build/` dir as well. What about passing a param into `_parseAssets()` - a directory to exclude?